src/assets/index.ts

```diff
@@ -0,0 +1,17 @@
+// generated file, do not modify its content
+import apps from './icons/apps.svg';
+import arrowUpLeft from './icons/arrow-up-left.svg';
+import checkmark from './icons/checkmark.svg';
+import cog from './icons/cog.svg';
+import download from './icons/download.svg';
+import folder from './icons/folder.svg';
+import heart from './icons/heart.svg';
+import home from './icons/home.svg';
+import link from './icons/link.svg';
+import night from './icons/night.svg';
+import pinned from './icons/pinned.svg';
+import returnKey from './icons/return-key.svg';
+import search from './icons/search.svg';
+import station from './icons/station.svg';
+const icons = {apps,arrowUpLeft,checkmark,cog,download,folder,heart,home,link,night,pinned,returnKey,search,station};
+export default icons;
```

Comment:
You can simplify this (and remove the default export):
```typescript
export { default as apps } from './icons/apps.svg';
export { default as arrowUpLeft } from './icons/arrow-up-left.svg';
etc...
```

Reply:
The file generated is now the same as the one in theme repo